### ISSUE TITLE
replicate.stream() gives blank spaces at the beginning of output stream

### ISSUE THREAD

**Issue by yosuaw** [replicate/replicate-python#277]
Hello, I got trouble when I want to stream the chat model output from replicate. I'm using replicate version 0.25.1. My code is look like this:
```
for event in replicate.stream(
    "meta/llama-2-7b-chat",
    input={
        "debug": False,
        "max_new_tokens": 800,
        "min_new_tokens": -1,
        "temperature": 0.75,
        "top_p": 0.95,
        "top_k": 50,
        "repetition_penalty": 1,
        "prompt": prompt,
        "system_prompt": SYSTEM_PROMPT,
        "min_new_tokens": -1,
        "prompt_template": "[INST] <<SYS>>\n{system_prompt}\n<</SYS>>\n\n{prompt} [/INST]",
    },
):
    print(str(event), end="", flush=True)
```

When I run it for the first time, it give me good result and desired output. But, for the 2nd run it gives me output with one blank spaces in the first line. For the 3rd run it gives me two blank spaces in the beginning of the output. Here's what my output looks like when I run the code 5th times (I got 4 blank spaces in the beginning of the output):
![image](https://github.com/replicate/replicate-python/assets/80568597/8305491f-1bf4-438b-a156-a47d84ac643e)

I don't know if it's caused by my prompt or bugs. Can someone help me with this?

**Comment by yosuaw** [comment]
I think it's not caused by my prompt. I tried to change the prompt and system prompt like this:
```
"prompt": "Hello how are you?",
"system_prompt": "Answer the question",
```
And I still faced the same issue:
![image](https://github.com/replicate/replicate-python/assets/80568597/ac6eeb22-c6f4-4583-a563-22facf82929d)

Any help or suggestion will be appreciated :)

**Comment by Pwntus** [comment]
Hi @yosuaw this should get fixed in coming versions. I the mean time - you can strip away leading newlines from the output as a quick fix.

**Comment by yosuaw** [comment]
Thank you so much for your response @Pwntus. Please let me know if the bug is fixed.

**Comment by mattt** [comment]
Hi @yosuaw. I believe this was the same issue as #287, which is now fixed in [0.25.2](https://github.com/replicate/replicate-python/releases/tag/0.25.2). Please update to the latest version of this library and let us know if that doesn't resolve your issue. Thanks!
